eos-updater-prepare-volume/eos-updater-prepare-volume

```diff
@@ -176,12 +176,26 @@ class VolumePreparer:
 
         return runtimes
 
-    def _add_autoinstall_flatpaks(self, flatpak_collection_refs):
-        applied_actions = EosUpdaterFlatpakInstaller.determine_flatpak_ref_actions_to_check(None)
-        for filename in applied_actions:
-            for action in applied_actions[filename]:
-                if action.type != EosUpdaterUtil.FlatpakRemoteRefActionType.UNINSTALL:
-                    flatpak_collection_refs.append((action.get_collection_id(), action.get_ref()))
+    def _get_autoinstall_flatpaks(self, flatpak_collection_refs):
```

Comment:
It seems like there's no longer any reason to have this argument with the way you've rewritten things

Reply:
Ah yes. :+1:

Reply:
Fixup pushed